giap.client/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+This file explains how Visual Studio created the project.
+
+The following tools were used to generate this project:
```

Comment:
Changelogs can be generated with releases in Github automatically. Its combines all the PR's and then it generates a changelog.